experimental/python/databricks/bundles/build.py

```diff
@@ -0,0 +1,479 @@
+import argparse
+import importlib
+import inspect
+import json
+import logging
+import os.path
+import sys
+from copy import deepcopy
+from dataclasses import dataclass, field, replace
+from typing import Callable, Optional, TextIO
+
+from databricks.bundles.core import Bundle, Diagnostics, Location, Resources
+from databricks.bundles.core._resource_mutator import ResourceMutator
+from databricks.bundles.core._transform import _transform
+from databricks.bundles.jobs import Job
+
+__all__ = []
+
+
+@dataclass
+class _Args:
+    phase: str
+    input: str
+    output: str
+    diagnostics: str
+    locations: Optional[str]  # FIXME should be required
+    unknown_args: list[str]
+
+
+@dataclass
+class _Conf:
+    resources: list[str] = field(default_factory=list)
+    mutators: list[str] = field(default_factory=list)
+    venv_path: Optional[str] = None
+
+    @classmethod
+    def from_dict(cls, d: dict) -> "_Conf":
+        known_keys = {"resources", "mutators", "venv_path"}
```

Comment:
You can use https://docs.python.org/3/library/dataclasses.html#dataclasses.fields to make this dynamic.

Reply:
Fixed in https://github.com/databricks/cli/pull/2486

Reply:
Thanks!